### PR TITLE
Update layout of expanded details content; fixes #192

### LINKED ISSUE #192: Layout of expanded details content
For the label/value pairs in the expanded detail panels, the spacing is too loose and affects readability.

- [x] This seems to work for the `<dt>` / `<dd>` pairs:
`<dt class="col-3 offset-1 col-md-2 offset-md-2">`
`<dd class="col-8">`

- [x] For "View in SearchWorks" and other standalone links or text:
`<div class="col-11 offset-1 col-md-11 offset-md-2">`

- [x] REMOVE `.justify-content-center` from the containing rows for both of the above. 

Outcomes:

detail layout phone
detail layout sm
detail layout md
detail layout lg

### PR DESCRIPTION
**Pull request by cbeer**
Fixes #192

Before:
Screen Shot 2019-07-22 at 07 42 12

After:
Screen Shot 2019-07-22 at 07 41 54

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -8,12 +8,12 @@ def active_page_class(name)
   # Wrap a link to the SearchWorks record for the given Catkey wrapped in the markup
   # necessary to be aligned with the content in the collapsible list sections
   def detail_link_to_searchworks(catkey)
-    content_tag(:div, class: 'row justify-content-center') do
-      content_tag(:div, class: 'col-5') do
+    content_tag(:div, class: 'row') do
+      content_tag(:div, class: 'col-11 offset-1 col-md-11 offset-md-2') do
```
Comment (mejackreed) [inline app/helpers/application_helper.rb]: I know this is what was specified, but should the `col-md-11` be a `col-md-10`?

```suggestion
      content_tag(:div, class: 'col-11 offset-1 col-md-10 offset-md-2') do
```

File: app/views/checkouts/_checkout.html.erb [hunk 2]
```diff
@@ -27,8 +27,8 @@
   </div>
   <div class="collapse w-100" id="collapseExample-<%= checkout.key.parameterize %>">
     <% if checkout.renewable? %>
-      <div class="row justify-content-center">
-        <div class="col-5">
+      <div class="row">
+        <div class="col-11 offset-1 col-md-11 offset-md-2">
```
Comment (mejackreed) [inline app/views/checkouts/_checkout.html.erb]:
```suggestion
        <div class="col-11 offset-1 col-md-10 offset-md-2">
```

File: app/helpers/application_helper.rb (hunk 1 above)
Comment (cbeer) [reply app/helpers/application_helper.rb]: @jvine ?

**Comment by cbeer** [comment]
(I'm going to regret not pulling out helper methods for these, I guess..)

**Comment by jvine** [comment]
Damn. Yes, my typo. This is correct: ` <div class="col-11 offset-1 col-md-10 offset-md-2">`

But I don't understand this comment:  

> col-md-2 offset-md-2 + col-8 = 13 cols

```
<dt class="col-3 offset-1 col-md-2 offset-md-2">
<dd class="col-8">
```
3 + 1 + 8 = 12, and 2 + 2 + 8 = 12

**Pull request merged**